jest.config.ts

```diff
@@ -12,6 +12,7 @@ const projects: any = fs
         rootDir: `packages/${name}`,
         displayName: name,
         moduleNameMapper: {
+            "@zk-kit/artifacts": ["<rootDir>/../../node_modules/@zk-kit/artifacts/dist/index.node.cjs"],
```

Comment:
as `@zk-kit/artifacts` is out of the monorepo packages, we must not override the corresponding entry it in the `moduleNameMapper` object. Is there a better way?